py/openage/convert/mediafile.py

```diff
@@ -327,6 +348,17 @@ def media_convert(args):
 
             media_files_extracted += 1
 
+    pool.close()
+    for i in media_out:
+        try:
+            i.wait()
+            # if anything was raised in child, it will be re-raised here
+            i.get()
+        except:
+            dbg("got exception", 1)
```

Comment:
Can you dump the rethrown exception here? Just `"got exception"` won't help debugging. Also we probably don't want to catch all exceptions here, not sure though.